ci/vscode.patch

```diff
@@ -696,10 +696,6 @@ index 0000000000..96fbd4b0bb
 +		}
 +
 +		const json = await response.json();
-+		if (!json.isLatest) {
```

Comment:
Wouldn't this cause no errors to be reported if an update fails?

Reply:
The response will only be a 200 if the update succeeded so it should already have bailed out on the line above. The problem with this line is that `isLatest` was always false since the currently running code was still the old version.

Although, that means I should log the status text and not just "unexpected response". Will do that right now.